sdks/android/Mentat/library/src/main/java/com/mozilla/mentat/Mentat.java

```diff
@@ -178,6 +231,15 @@ public EntityBuilder entityBuilder(long entid) {
         return null;
     }
 
+    /**
+     * Creates a new transaction ({@link InProgress}) and returns an {@link EntityBuilder} for a new
+     * entity with `tempId` for that transaction.
+     *
+     * TODO: Throw an exception if the result contains an error.
```

Comment:
Do we have a ticket on file for this error handling discipline?  Let's link to it, please.

Reply:
https://github.com/mozilla/mentat/issues/700